### DISCUSSION TITLE
youtube域名和IP地址集

### DISCUSSION THREAD

**Discussion by iovefish** [Openwrt-Passwall/openwrt-passwall2#191] [General]
哪位兄弟有YouTube的域名和IP地址集，用于passwall2分流，谢谢了

**Comment by motiyb** [comment]
Proxy里面的应该已经包含了呀。我的proxy指定代理后，油管就能上了。

**Comment by wangdaye078** [comment]
geosite:youtube
其实包含了geosite:google里面就有youtube
